.github/workflows/main.yml

```diff
@@ -0,0 +1,67 @@
+name: CI
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the master branch
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install Bazelisk
+        run: |
```

Comment:
Question: Does this list of commands actually fail on error on each individual commands? (e.g. `set -e` in bash) Or does it simply continue and exits with the last command exit status? (default in Bash)

I hope GitHub does a `set -e` by default... can't really find it, though.